### PR TITLE
[18.01] replace incorrect list add syntax

### PR DESCRIPTION
**Pull request by martenson**
Turns out python lists do not have this handy method...🤔

credit to @richard-burhans

### CODE REVIEW

**Comment by dannon** [comment]
Lists have `.append`, sets have `.add`.  The first line of this change looks correct, but the last three will break, I think.

**Comment by martenson** [comment]
@dannon Good point, that also explains why I made this mistake in the first place.

I pushed a fix.

File: lib/galaxy/managers/libraries.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def list(self, trans, deleted=False):
             allowed_library_manage_ids = set([])
             for action in all_actions:
                 if action.action == library_access_action:
-                    accessible_restricted_library_ids.add(action.library_id)
+                    accessible_restricted_library_ids.append(action.library_id)
```
Comment (nsoranzo) [inline lib/galaxy/managers/libraries.py:148]: Or change also `accessible_restricted_library_ids` to a set?

Comment (martenson) [reply lib/galaxy/managers/libraries.py:148]: I vaguely remember there was a reason for this at one point. I don't think that is a bug though (even if not), so if you feel inclined please open a PR against dev.

Comment (mvdbeek) [reply lib/galaxy/managers/libraries.py:148]: Not a bug, but `in` (and I suppose also sqlalchemy's `in_`) should be a lot faster for big sets than big lists.

Comment (nsoranzo) [reply lib/galaxy/managers/libraries.py:148]: The actual code comes from https://github.com/galaxyproject/galaxy/pull/5542

Comment (martenson) [reply lib/galaxy/managers/libraries.py:148]: I am all for speed, but at this time of imminent release I do not want to wait for another round of tests. I opened a PR with this suggestion against dev in https://github.com/galaxyproject/galaxy/pull/5681.

**Pull request merged**